resources/java/classes/Records.java

```diff
@@ -0,0 +1,13 @@
+public record Person(String name, String address) {
+    public Person(String name) {
+        this(name, "Unknown");
+    }
+}
+public record Mensch (String name, String address) {
+    public Person {
```

Comment:
Have you tried to compile that code? I guess that it should not compile to have a compact constructor for "Person" in a record "Mensch"? 

You may also change the name "Mensch" to an english term like "human" or "employee" or whatever english word you like :)

Reply:
you are right, it could not be compiled. I have checked all java test resources on compile-ability and adjusted them afterwards.